ext/drcovlib/modules.c

```diff
@@ -663,13 +669,23 @@ drmodtrack_offline_read(file_t file, const char *map, OUT const char **next_line
                     goto read_error;
                 info->mod[i].containing_id = mod_id;
                 buf = skip_commas_and_spaces(buf, 4);
-            } else {
+            }
+            else if (version == 3) {
```

Comment:
minor nit: since already changing some other things I would suggest putting the else on the prior line for consistency w/ the surrounding if-else chains

Reply:
Done.